src/index.ts

```diff
@@ -1,8 +1,8 @@
 // Types
-export * from './core/types';
+export type * from 'alpha-ai-avatar-sdk-js';
```

Comment:
suggestion: I think you need to remove `type` otherwise clients of the sdk won't be able to use the enums, right @Wrsanches?

Reply:
@gfpacheco great suggestion 👍🏾